packages/client/hmi-client/src/components/models/tera-model-diagram.vue

```diff
@@ -128,6 +147,7 @@ const props = defineProps<{
 	model: Model | null;
 	isEditable: boolean;
 	nodePreview?: boolean;
+	strataView?: boolean;
```

Comment:
My feel is that you are overloading too much functionalities into tera-model-diagram. 

It becomes difficult to understand, and to use because the number of combinations you need to be aware of grows geometrically. For example you don't need a math-editor for strata, but it is included anyway .... etc etc

We had something similar before (albeit with API) that takes in like 5-6 different options, some mutually exclusive and some complementary,  and it is very tedious to get things right. So I think an independent strata-component would be better suited.

Reply:
Good point